Magical Nights/items/enchanted.json

```
@@ -4,20 +4,60 @@
     "type": "GENERIC",
     "symbol": "/",
     "color": "brown",
+    "name": { "str": "minor staff of the magi", "str_pl": "minor staves of the magi" },
+    "description": "A simple carved staff, made of wood.  It faintly glows with magic when you cast spells.  Wielding it slightly increases your mana regeneration",
+    "price": "100 USD",
+    "material": [ "wood" ],
+    "weapon_category": [ "QUARTERSTAVES" ],
+    "flags": [ "SHEATH_SPEAR", "ALWAYS_TWOHAND","FRAGILE_MELEE", "MAGIC_FOCUS" ],
+    "relic_data": {
+      "passive_effects": [ { "has": "WIELD", "condition": "ALWAYS", "values": [ { "value": "MANA_CAP", "multiply": 0.5 } ] } ]
+    },
+    "weight": "1400 g",
+    "volume": "3 L",
+    "category": "weapons",
+    "to_hit": 3,
+    "bashing": 17
+  },
+  {
+    "id": "magi_staff_lesser",
+    "type": "GENERIC",
+    "symbol": "/",
+    "color": "brown",
```

```suggestion
    "color": "brown",
    "looks_like": "magi_staff_minor",
```
This way it's not jarringly  ascii while using the tileset (though of course, ideally I'll give it a somewhat unique sprite/combination in the future.)